test/test_manifest.json

```diff
@@ -5907,5 +5907,14 @@
        "link": true,
        "forms": true,
        "type": "eq"
+    },
+    {  "id": "issue14021",
+       "file": "pdfs/issue14021.pdf",
+       "md5": "d18aa84135ce985c70a8f56306ecb95f",
+       "link": true,
+       "form": true,
+       "rounds": 1,
+       "lastPage": 1,
```

Comment:
As far as I can tell, there's actually no form-elements on that page.
I'm guessing that you actually want `firstPage: 2` and `lastPage: 2` here?